### ISSUE TITLE
Implement BeamGroupViewModel

### ISSUE THREAD

**Issue by jsbean** [jsbean/DNM#172]
```Swift
public class BeamGroupViewModel: DurationNode {
    public var model: DurationNode

    public init(model: DurationNode) {  }
}
```

**Comment by jsbean** [comment]
```Swift
public class BeamGroup {
    public var viewModel: BeamGroupViewModel
}
```

**Comment by jsbean** [comment]
Closed with e3c078e2f6dc6030ec29f99ced69b8b7b1333e6f
